### PR TITLE
Add callout for important force reset  notice

### PR DESCRIPTION
**Pull request by antoniofilipovic**
### Description

Add docs for following PR:https://github.com/memgraph/memgraph/pull/2135

### Pull request type

Please check what kind of PR this is:

- [x] Fix or improvement of an existing page
- [ ] New documentation page, release related

### Related PRs and issues

PR this doc page is related to: 
(especially necessary if the PR is related to a release)

Closes:
[(paste the link to the issue it closes)](https://github.com/memgraph/memgraph/pull/2135)

### Checklist:

- [x] Check all content with Grammarly
- [x] Perform a self-review of my code
- [x] Make corresponding changes to the rest of the documentation (consult with the DX team)
- [x] The build passes locally
- [x] My changes generate no new warnings or errors
- [x] Add a corresponding label
- [x] If release-related, add a product and version label
- [x] If release-related, add release note on product PR

### CODE REVIEW

File: pages/clustering/high-availability.mdx [hunk 1]
```diff
@@ -361,10 +361,15 @@ the cluster enters a state of force reset where the cluster is reset to the stat
...
 the RPC request to a REPLICA instance to promote itself to MAIN can succeed, but writing to the Raft log that the instance was promoted can fail. 
-Force reset includes demoting every alive instance to REPLICA, and executing the failover procedure once again. Such a procedure is needed as currently cluster doesn't track 
+Force reset includes demoting every alive instance to REPLICA, and executing the failover procedure once again. Such a procedure is needed as of this moment cluster doesn't track 
 where the action failed exactly, but only whether it fully succeded. Raft log is taken as a source of truth at all times. In case the leader coordinator dies while executing 
 the force reset, the next coordinator which is elected as the leader, will continue executing the force reset. Action is executed until it succeeds.
 
+<Callout type="info">
+It is important to note that in case some action fails and all instances are down, the leader will try to execute force reset until one instance is promoted to MAIN. Until then no other action is allowed
+on cluster.
+</Callout>
```
Comment (kgolubic) [inline pages/clustering/high-availability.mdx:371]: Replace text in Callout with:

It is important to note that if an action fails and all instances are down, the leader will attempt to execute a force reset until one instance is promoted to MAIN. Until then, no other actions are allowed on the cluster.

Add one blank line above and below this text (within Callout tags).

**Review by kgolubic — CHANGES_REQUESTED** [review]
Once the text is updated it will be good to go.

**Review by hal-eisen-MG — APPROVED** [review]
LGTM

**Review by kgolubic — APPROVED** [review]
Everything looks OK.

**Pull request merged**